apps/prairielearn/python/prairielearn/misc_utils.py

```diff
@@ -18,7 +18,7 @@
 
 
 def iter_keys() -> Generator[str, None, None]:
-    """A continuous alphabetic list of the form ['a', 'b', ..., 'z', 'aa', 'ab', ..., 'zz', 'aaa', 'aab', ...].
+    """A continuous alphabetic list of the form `['a', 'b', ..., 'z', 'aa', 'ab', ..., 'zz', 'aaa', 'aab', ...]`.
```

Comment:
```suggestion
    """A continuous alphabetic list of the form `["a", 'b', ..., 'z', 'aa', 'ab', ..., 'zz', 'aaa', 'aab', ...]`.
```
Should change here and in the next function.

Reply:
I can think of a ton of other spots where we aren't consistent on the docs between single and double quotes. We should try to fix them all, or none of them IMO.

Reply:
What do you think @eliotwrobson ?

Reply:
I don't think it's a huge deal either way. If it's still pretty inconsistent anyway, no reason to change right now.

Reply:
I think i'll wait to resolve these then.

Reply:
It would be great if there was an automated way of doing this. Otherwise, it's probably not worth the time / effort it would take to go over everything.